guides/common/assembly_converting-a-host-with-convert2rhel.adoc

```diff
@@ -0,0 +1,50 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly_converting-a_host-with-convert2rhel_{context}"]
+= Converting a Host to {RHEL} with Convert2RHEL
+
+Convert2RHEL enables the conversion of Red Hat Enterprise Linux derivative distributions into a supportable {RHEL} state while retaining existing applications and configurations.
+
+The conversion process is similar to a minor release upgrade of {RHEL} in which every RPM package on the system is replaced.
+Third-party packages and non-{Team} packages not available in {RHEL} are not replaced.
+
+Convert2RHEL removes unnecessary packages such as logos, or packages known to cause issues during the conversion, and replaces the `CentOS-release` package with the `rhel-release` package.
```

Comment:
Question: Does this only work from "centos" to "rhel"? What about Oracle Linux, Alma Linux, and Rocky Linux, which are binary/bug compatible with RHEL?

Reply:
I'm told they are not included  in this release. They'll be included at or after 7.0